react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+export default function ValidatedInput() {
+  const [isPassword, setIsPassword] = useState('');
+  const [tooShort, setTooShort] = useState(false);
```

Comment:
You shouldn't declare this as state since it can be computed from your other state, `password`. In general, if a value can be computed from state, then it is not state and should be computed in the function body.